packages/docs/layouts/default.vue

```diff
@@ -60,6 +69,7 @@ useHead({
   link: [
     { href: 'https://cdn.jsdelivr.net/npm/@mdi/font@5.9.55/css/materialdesignicons.min.css', rel: 'stylesheet' },
     { href: 'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css', rel: 'stylesheet' },
+    { href: "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css", rel: "stylesheet" },
```

Comment:
@RVitaly1978,  Can't we use icon from Material Design Icons? This loads tons of extra icons.